src/main/java/Action.java

```diff
@@ -0,0 +1,150 @@
+import java.lang.reflect.Array;
+import java.util.*;
+
+public class Action {
+    private String commandName;
+    private Item item;
+    private Room room;
+
+    public Action(String commandName, Item item, Room room) {
+        this.commandName = commandName;
+        this.item = item;
+        this.room = room;
+    }
+
+    public static String printInventory(ArrayList<Item> playerItems) {
+        String playerInventory = "";
+        String playerInventoryFinal;
+
+        for (Item item : playerItems){
+            playerInventory = item.getNameFromItem() + ", " + playerInventory;
+        }
+        int unnecessaryComma = playerInventory.lastIndexOf(",");
```

Comment:
I am pretty sure there is a cleaner way of doing this without using this variable and having both a playerInventory and a playerInventoryFinal variable.